packages/synapse-react-client/src/components/GenericCard/SynapseCardLabel.tsx

```diff
@@ -181,9 +181,7 @@ export const SynapseCardLabel: React.FC<SynapseCardLabelProps> = props => {
         <p>
           {split.map((el, index) => {
             const { baseURL, URLColumnName, wrapValueWithParens } = labelLink
-            const elOrRowId = labelLink.overrideValueWithRowID
-              ? `syn${rowId}`
-              : el
```

Comment:
It seems like we lose the `syn` here if this is a view. As far as I can tell, `overrideValueWithRowID` is only used for the ARK Portal Datasets explore page. It seems like these links will be different, but I think both the new and old links will still work. For example, these both seem to resolve to the same details page, making this a non-breaking change:

`https://staging.arkportal.synapse.org/Explore/Datasets/DetailsPage?id=syn29837062`

`https://staging.arkportal.synapse.org/Explore/Datasets/DetailsPage?id=29837062`

Reply:
The only time rowId is passed into the SynapseCardLabel is from SynapseTable, and that rowId always has the 'syn' prefix (if it exists)

Reply:
Hold on, my assumption that the rowId is only a string when using the new configuration values is wrong.  Let me look into this...